Cannot find module 'lodash' (Windows 10)

### Description of the bug/issue

When I install Nightwatch 2.6.2 on Windows 10 and attempt to run Nightwatch I receive an error 

```bash
An error occurred while trying to start the Nightwatch Runner:                           │
│   Cannot find module 'lodash'
```

### Steps to reproduce

Windows 10
1. Open powershell prompt in a new/empty directory
2. Run `npm install nightwatch`
3. Run `npx nightwatch`

```
┌────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                                                            │
│   An error occurred while trying to start the Nightwatch Runner:                           │
│   Cannot find module 'lodash'                                                              │
│   Require stack:                                                                           │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\reporter\results.js       │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\reporter\index.js         │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\testsuite\index.js        │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\runner\matchers\tags.js   │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\runner\folder-walk.js     │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\runner\runner.js          │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\runner\cli\cli.js         │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\lib\index.js                  │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\bin\runner.js                 │
│   - C:\Users\Mr\Desktop\nw supertest\node_modules\nightwatch\bin\nightwatch                │
│                                                                                            │
│                                                                                            │
└────────────────────────────────────────────────────────────────────────────────────────────┘
```

### Sample test

Occurs before testing can begin



### Command to run

```bash
mkdir exampleDir
cd exampleDir
npm install nightwatch
npx nightwatch
```


### Verbose Output

```fundamental
n/a
```


### Nightwatch Configuration

```node
n/a
```


### Nightwatch.js Version

2.6.2

### Node Version

16.15.0

### Browser

Chrome 108

### Operating System

Windows 10

### Additional Information

_No response_

--- a/lib/reporter/results.js
+++ b/lib/reporter/results.js
@@ -1,5 +1,5 @@
 const AssertionError = require('assertion-error');
-const {merge: lodashMerge} = require('lodash');
+const lodashMerge = require('lodash.merge');
 const Utils = require('../utils');
 const {Logger} = Utils;
 

--- a/test/extra/commands/customCommandAppendResults.js
+++ b/test/extra/commands/customCommandAppendResults.js
@@ -1,0 +1,14 @@
+module.exports = class AppendResult {
+  command() {
+    const {name} = this.client.reporter.currentTest;
+    const {testResults} = this.client.reporter;
+    testResults.appendTestResult({
+      [name]: {
+        customReport: {
+          success: true
+        }
+      }
+    });
+
+  }
+};

--- a/test/sampletests/appendtestresult/sampleWithAppendResults.js
+++ b/test/sampletests/appendtestresult/sampleWithAppendResults.js
@@ -1,0 +1,5 @@
+module.exports = {
+  demoTest(client) {
+    client.url('http://localhost').customCommandAppendResults().end();
+  }
+};

--- a/test/src/runner/testReporter.js
+++ b/test/src/runner/testReporter.js
@@ -329,4 +329,39 @@ describe('testReporter', function() {
     assert.strictEqual(possibleError, null);
 
   });
+
+  it('check reporter output with appended results', async function() {
+    let possibleError = null;
+    const testPath = [path.join(__dirname, '../../sampletests/appendtestresult/sampleWithAppendResults.js')];
+    const customCommands = [path.join(__dirname, '../../extra/commands')];
+
+    try {
+      await runTests({
+        source: testPath
+      }, settings({
+        custom_commands_path: customCommands,
+        globals: {
+          waitForConditionPollInterval: 20,
+          waitForConditionTimeout: 50,
+          retryAssertionTimeout: 50,
+          reporter: function (results) {
+            const module = results.modules['sampleWithAppendResults'];
+            
+            assert.ok(Object.keys(module).includes('completedSections'));
+            const completedSections = module['completedSections'];
+            assert.ok(Object.keys(completedSections).includes('demoTest'));
+
+            const test = completedSections['demoTest'];
+            assert.ok(Object.keys(test).includes('customReport'));
+            assert.deepStrictEqual(test.customReport, {success: true});
+          }
+        },
+        silent: true,
+        output: false
+      }));
+    } catch (err) {
+      possibleError = err;
+    }
+    assert.strictEqual(possibleError, null);
+  });
 });

